packages/labs/compiler/package.json

```diff
@@ -1,18 +1,78 @@
 {
   "name": "@lit-labs/compiler",
+  "description": "Compiler to prepare Lit templates at build time",
   "version": "0.0.0",
   "private": true,
-  "description": "Lit compiler to prepare Lit templates at build time",
   "author": "Google LLC",
   "homepage": "https://github.com/Lit/Lit/tree/main/packages/labs/compiler",
   "license": "BSD-3-Clause",
   "type": "module",
   "repository": {
     "type": "git",
     "url": "https://github.com/Lit/Lit.git",
-    "directory": "packages/compiler"
+    "directory": "packages/labs/compiler"
   },
   "bugs": {
     "url": "https://github.com/lit/lit/issues"
+  },
+  "scripts": {
+    "build:ts": "wireit",
```

Comment:
Should this be added to the monorepo root's package.json as a dependency to "build:ts"?

This package might not need it but do want a "build" script that depends on "build:ts", also added to the monorepo root's, so `npm run build` from monorepo root builds everything?

Reply:
That is a good idea – I'll split it into a separate change.